src/index.html

```diff
@@ -5,8 +5,459 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page-content">
+    <header class="header">
+      <a href="#home">
+        <img
+         src="images/logo.png"
+         alt="logo Moyo"
+         class="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="list">
+          <li class="list-item">
+            <a href="#apple" class="is-active">
+              Apple
+            </a>
+          </li>
+
+          <li class="list-item">
+            <a href="#samsung" class="link">
+              Samsung
+            </a>
+          </li>
+
+          <li class="list-item">
+            <a href="#smartphones" class="link">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="list-item">
+            <a
+              href="#laptops_computers"
+              data-qa="nav-hover"
+              class="link"
+             >
+             Laptops & Computers
+            </a>
+          </li>
+
+          <li class="list-item">
+            <a href="#gadgets" class="link">
+              Gadgets
+            </a>
+          </li>
+
+
+          <li class="list-item">
+            <a href="#tablets" class="link">
+              Tablets
+            </a>
+          </li>
+
+          <li class="list-item">
+            <a href="#photo" class="link">
+              Photo
+            </a>
+          </li>
+
+          <li class="list-item">
+            <a href="#video"class="link">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
```

Comment:
According to the rules BEM
nav>nav__list>nav__item>nav__link! Change it please for all page